stetho-sample/src/debug/java/com/facebook/stetho/sample/APODContentProviderDumper.java

```diff
@@ -0,0 +1,112 @@
+package com.facebook.stetho.sample;
+
+import android.content.ContentResolver;
+import android.database.Cursor;
+import com.facebook.stetho.dumpapp.DumpException;
+import com.facebook.stetho.dumpapp.DumperContext;
+import com.facebook.stetho.dumpapp.DumperPlugin;
+
+import java.io.PrintStream;
+import java.util.List;
+
+public class APODContentProviderDumper implements DumperPlugin {
+
+  private static final String NAME = "content";
+
+  private static final String CMD_LIST = "list";
+  private static final String USAGE_LIST = CMD_LIST;
+
+  private static final String CMD_CLEAR = "clear";
+  private static final String USAGE_CLEAR = CMD_CLEAR;
+
+  private static final String CMD_REMOVE_ROW = "delete";
+  private static final String USAGE_REMOVE_ROW = CMD_REMOVE_ROW + " <row id>";
```

Comment:
IMO, it's better to just put this extra information in the help directly.  It's not repeated anywhere else...